lib/Mojo/File.pm

```diff
@@ -63,17 +63,18 @@ sub list_tree {
 
   # The File::Find documentation lies, this is needed for CIFS
   local $File::Find::dont_use_nlink = 1 if $options->{dont_use_nlink};
-
+  
```

Comment:
Why does this empty line change?

Reply:
Not sure how that whitespace crept in - now removed